src/pages/Swap/AdvancedSwapMode/PairDetails/PairDetails.component.tsx

```diff
@@ -16,13 +16,13 @@ interface PairDetailsProps {
   activeCurrencyOption: Token | null | undefined
 }
 
-const RelativePrice = ({ relativePrice }: { relativePrice: string }) => {
-  if (Number(relativePrice) > 1) return <>{relativePrice}</>
+const DoubleSizedPrice = ({ price }: { price: string }) => {
```

Comment:
maybe `DoubleDigitPrice` is a better name.

Reply:
This price has 2 different sizes, one for up to 6 digits, and another size for the remaining digits.